modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/AMQPUtils.java

```diff
@@ -71,6 +71,8 @@
 
     public static String TOPIC_EXCHANGE_NAME = "amq.topic";
 
+    public static String PROTOCOL_TYPE = "AMQP";
+
```

Comment:
There is an enum defined just for this (ProtocolType), should have used it instead of defining a String here.